README.md

```diff
@@ -1,3 +1,21 @@
-# Batch to Compiler [![Build Status](https://travis-ci.com/enieber/batch.svg?branch=master)](https://travis-ci.com/enieber/batch)
+# Batch [![Build Status](https://travis-ci.com/enieber/batch.svg?branch=master)](https://travis-ci.com/enieber/batch)
 
-This project is rank to compiler based in the Batch lote process.
+## About this Project
+
+This is a platform for queue for process and activities.
+Batch should make your life easier when the subject is about organization and comunication.
+
+## TODO
+
+Well, this project is in a initial step of development, we plan to release _ASAP_.
+Currently we're working in backend and front-end at the same time.
+
+### I'll show you some of our plans:
+
+| Plan                         | Done?               |
+| ---------------------------- | ------------------- |
+| Make Brazil greate again? 😂 | [ ]                 |
+| Document the Api             | [ ] (_In progress_) |
```

Comment:
maybe need to reduce in minor steps like:

- [x] list process 
- [x] add new process
- [x] change position of process
- [ ] edit process
- [ ] remove project

Reply:
This is just a General Talk about the project, not a detailed one